File: easy/array/plusone.cpp
vector<int> plusOne(vector<int>& digits){
	int n = digits.size();
	for(int i = n-1; i >= 0; --i){
		if(++digits[i] < 10)
			return digits;
		digits[i] = 0;
	}
	digits[0] = 1; // if digits = {9,9,9}
	digits.push_back(0);
	return digits;
}

File: easy/array/plusone.py
def plusone(digits):
	n = len(digits)
	for x in range(n-1, -1, -1):
		digits[x] += 1
		if digits[x] < 10:
			return digits
		digits[x] = 0
	
	digits[0] = 1 # if digits = {9,9,9}
	digits.append(0)
	return digits		